source/features/quick-repo-deletion.tsx

```diff
@@ -107,7 +105,6 @@ async function start(buttonContainer: HTMLDetailsElement): Promise<void> {
 			{action: false},
 		);
 		select('.application-main')!.remove();
-		await cache.delete(getCacheKey());
```

Comment:
I think this is the only external usage you mentioned, which was just a cache shortcut for `forked-to` itself

Reply:
You are correct, what I was thinking was the old way we had this implemented that `show-open-prs-on-forks` relied on this.